src/index.js

```diff
@@ -20,15 +19,97 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const initState = {
+  result: 0,
+  num1: '',
+  operator: '',
+  num2: '',
+};
+
+function render(resultState) {
+  const numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0];
+  const operators = ['+', '-', '*', '/', '='];
+
+  const calculate = (state) => ({
+    '+': Number(state.num1) + Number(state.num2),
+    '-': Number(state.num1) - Number(state.num2),
+    '*': Number(state.num1) * Number(state.num2),
+    '/': Number(state.num1) / Number(state.num2),
+  })[state.operator];
+
+  const calculator = (state, item) => {
+    const result = calculate(state);
```

Comment:
필요한 값만 사용하는건 어떨까요? 단순히 연산에 대한 객체에 현재 선택된 연산자와 값들을 사용하는거죠.
```suggestion
const calculate = {
  '+': (x, y) => x + y,
  '-': (x, y) => x - y,
  '*': (x, y) => x * y,
  '/': (x, y) => x / y
};

const calculator = ({ operator, num1, num2 }) => {  // 이 경우에는 destructuring(구조분해할당)을 사용하는 것도 좋겠네요.
    const result = calculate[operator](num1, num2);
    ...
```

Reply:
계산 로직을 간단히하고 디스트럭쳐링을 이용해 객체의 어떤 프로퍼티를 사용하는지 좀더 명확하게 만들어보겠습니다. 감사합니다!